tests/unit/amo/components/TestInstallWarning.js

```diff
@@ -0,0 +1,346 @@
+          addonIsRecommended: false,
+          experimentEnabled: true,
+          installStatus: INSTALLED,
+        }),
+      ).toEqual(false);
+    });
+
+    it('returns false if the experiment is disabled', () => {
+      expect(
+        couldShowWarning({
+          addonIsExtension: true,
+          addonIsRecommended: false,
+          experimentEnabled: false,
+          installStatus: UNINSTALLED,
+        }),
+      ).toEqual(false);
+    });
+  });
+
+  it('sets a dimension on mount if a variant exists', () => {
+    const _tracking = createFakeTracking();
+    const variant = VARIANT_INCLUDE_WARNING;
+
+    render({ _tracking, variant });
+
+    sinon.assert.calledWith(_tracking.setDimension, {
+      dimension: INSTALL_WARNING_EXPERIMENT_DIMENSION,
+      value: variant,
+    });
+  });
+
+  it('does not set a dimension on mount if a variant does not exist', () => {
+    const _log = getFakeLogger();
+    const _tracking = createFakeTracking();
+
+    render({ _log, _tracking, variant: undefined });
+
+    sinon.assert.notCalled(_tracking.setDimension);
+    sinon.assert.calledWith(
+      _log.debug,
+      `No variant set for experiment "${EXPERIMENT_ID}"`,
+    );
+  });
+
+  it('passes installStatus to couldShowWarning on mount', () => {
+    const _couldShowWarning = sinon.spy();
+    const isRecommended = false;
+    const experimentEnabled = true;
+    const installStatus = UNINSTALLED;
+    const addon = { ...fakeAddon, is_recommended: isRecommended };
+    _setInstallStatus({ addon, status: installStatus });
+
+    render({
+      _couldShowWarning,
+      addon: createInternalAddon(addon),
+      experimentEnabled,
+      variant: VARIANT_INCLUDE_WARNING,
+    });
+
+    sinon.assert.calledWith(_couldShowWarning, {
+      addonIsExtension: true,
+      addonIsRecommended: isRecommended,
+      experimentEnabled,
+      installStatus,
+    });
+  });
+
+  it('passes installStatus to couldShowWarning on update', () => {
+    const _couldShowWarning = sinon.spy();
+    const isRecommended = false;
+    const experimentEnabled = true;
+    const installStatus = UNINSTALLED;
+    const addon = { ...fakeAddon, is_recommended: isRecommended };
+
+    const root = render({
+      _couldShowWarning,
+      addon: createInternalAddon(addon),
+      experimentEnabled,
+      variant: VARIANT_INCLUDE_WARNING,
+    });
+
+    sinon.assert.calledWith(_couldShowWarning, {
+      addonIsExtension: true,
+      addonIsRecommended: isRecommended,
+      experimentEnabled,
+      installStatus: undefined,
+    });
+
+    _couldShowWarning.resetHistory();
+
+    root.setProps({ installStatus });
+
+    sinon.assert.calledWith(_couldShowWarning, {
+      addonIsExtension: true,
+      addonIsRecommended: isRecommended,
+      experimentEnabled,
+      installStatus,
+    });
+  });
+
+  describe('display tracking event', () => {
+    const _tracking = createFakeTracking();
+    const variant = VARIANT_INCLUDE_WARNING;
```

Comment:
I'd suggest you create a `describe` block for `VARIANT_INCLUDE_WARNING` (inside the `describe` block for `display tracking event`).